mobile-browser-based-version/src/components/sidebar/Settings.vue

```diff
@@ -191,7 +191,7 @@ export default defineComponent({
       'setPlatform'
     ]),
     toggleIndexedDB () {
-      this.setIndexedDB(!this.$store.state.useIndexedDB && window.indexedDB)
+      this.setIndexedDB(!!(!this.$store.state.useIndexedDB && window.indexedDB))
```

Comment:
the commit function in `store.js` already converts its argument to boolean, to avoid writing this trick in every call